pillar/heroku/ocw-studio.sls

```diff
@@ -51,6 +53,8 @@
       'GITHUB_WEBHOOK_BRANCH': 'release',
       'MAILGUN_SENDER_DOMAIN': 'ocw.mail.odl.mit.edu',
       'OCW_STUDIO_BASE_URL': 'https://ocw-studio.odl.mit.edu/',
+      'OCW_STUDIO_DRAFT_URL': '???',
+      'OCW_STUDIO_LIVE_URL': 'https://ocwnext.odl.mit.edu/',
```

Comment:
Assuming `OCW_STUDIO_LIVE_URL` should equal the production URL, not sure what `OCW_STUDIO_DRAFT_URL` should be

Reply:
Where are these being used? Is this for the pipeline to set the base URL in the rendered HTML? The DRAFT_URL is whatever we are using as the preview domain. Also, we should have different values for RC vs production, and currently ocwnext.odl is the output of the single build server. Might be worth getting @pdpinch 's opinion on what we want the URL to be going forward, at least until we make the final cutover to ocw.mit

Reply:
Ignore the RC vs prod part of the comment, just looked at this in context.

Reply:
These are the URL's where the draft and live outputs will be, which are used in the email notifications sent to site admins:  https://user-images.githubusercontent.com/187676/125663229-878bd030-a275-4d27-9b4a-86e14b6cc692.png